platform/solo5/librumpnet_ukvmif/ukvmif_user.c

```diff
@@ -125,12 +128,20 @@ int
 VIFHYPER_CREATE(const char *devstr, struct virtif_sc *vif_sc, uint8_t *enaddr,
 	struct virtif_user **viup)
 {
+	solo5_result_t ret;
 	struct virtif_user *viu = NULL;
 
+	if (net_handle == 0) {
```

Comment:
Don't think the check is needed. And if it is, it means that this function is called multiple times, and that would lead to `viu` being allocated more than once.

Reply:
Agree. It should be called only once. Fixed in c71a670a1ddb6d7f998eedde6563ba57b3b7b5d4.